cli/src/commands/func/stop.rs

```diff
@@ -0,0 +1,85 @@
+use crate::client::Client;
+use crate::error::Error;
+use crate::function::Function;
+use crate::project::Project;
+use eyre::{Context, Result};
+use http::StatusCode;
+use kinetics_parser::Parser;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct StopRequest {
+    pub project_name: String,
+    pub function_name: String,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct StopResponse {
+    /// Datetime when throttling was applied
+    pub throttled_at: String,
+
+    /// The reason for throttling,
+    /// e.g. user request or account limit.
+    pub reason: String,
+}
+
+/// Applies throttling to a function.
+/// The function stop receiving requests.
```

Comment:
@quesabe I can't find the exact style guide, but from what I remember the title and the message of rustdoc comment should be separated with an empty line.
https://doc.rust-lang.org/rust-by-example/meta/doc.html